### ISSUE TITLE
0x10 algo won't mine but will benchmark

### ISSUE THREAD

**Issue by derekdal7** [andru-kun/wildrig-multi#125]
hi, I've looked around and tried several things.  the miner will start with 0x10 but will never show hashrate.  it does show on the pool side as a client for my wallet but without hashrate.  I tried to run benchmark and it worked.  trying to use with with zerg

**Comment by andru-kun** [comment]
strange, what hpubare you using? And what hashrate do you get in benchmark mode?

**Comment by derekdal7** [comment]
yeah I thought it was strange also.  I get 17609Kh on 3070ti.  its down tuned a bit for better power use, I'm sure it could get more.  I should also add that the GPU load is 0 in gpu-z when pool mining but in benchmark its 95%.  running 512.x drivers in W10

**Comment by andru-kun** [comment]
oh, looks like your url to pool is wrong. Check if you use ssl port, if so, don't forget to use stratum+tcps:// prefix. "s" in it is very important. Also zerg can send job with some delay and you need to wait, or just use different pool

**Comment by derekdal7** [comment]
I've looked at those items, thx.  guess I'll continue looking when I have time
